kustomize/README.md

```diff
@@ -0,0 +1,176 @@
+# Use Kustomize to deploy Online Boutique with variants
+
+This page walks you through the deployment options for the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) sample application on a [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine) cluster using Kustomize. Each deployment variant is designed as a [**Kustomize Component**](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/components.md), so you can enable multiple variants at once.
+
+## What is Kustomize?
+
+Kustomize is a Kubernetes configuration management tool that allows users to customize their manifest configurations without duplication. Its commands are built into `kubectl` as `apply -k`. This repo uses Kustomize to enable Online Boutique deployment variants; a complete list of variants can be found at [/README.md](https://github.com/GoogleCloudPlatform/microservices-demo#other-deployment-options). More information on Kustomize can be found on the [official Kustomize website](https://kustomize.io/).
```

Comment:
Hyperlink Kustomize (https://kustomize.io/)

Reply:
This sentence is not necessary, as the section it's referring to is just below it.

"This repo uses Kustomize to enable Online Boutique deployment variants; a complete list of variants can be found at [/README.md](https://github.com/GoogleCloudPlatform/microservices-demo#other-deployment-options)."